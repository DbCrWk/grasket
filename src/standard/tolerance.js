// @flow

export const Tolerance = 0.001;
export default Tolerance;
