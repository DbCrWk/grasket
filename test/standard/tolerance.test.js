// @flow
import tolerance from '#src/standard/tolerance';

describe('tolerance', () => {
    it('remains a standard', () => {
        expect.assertions(1);

        expect(tolerance).toMatchSnapshot();
    });
});
