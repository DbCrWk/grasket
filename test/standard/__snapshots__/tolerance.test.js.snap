// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tolerance remains a standard 1`] = `0.001`;
